Godeps/Godeps.json

```diff
@@ -1,15 +1,25 @@
 {
 	"ImportPath": "github.com/kubernetes-incubator/cri-containerd",
 	"GoVersion": "go1.7",
-	"GodepVersion": "v79",
+	"GodepVersion": "v74",
 	"Packages": [
 		"./..."
 	],
 	"Deps": [
 		{
+			"ImportPath": "github.com/Sirupsen/logrus",
+			"Comment": "v0.11.0",
+			"Rev": "d26492970760ca5d33129d2d799e34be5c4782eb"
+		},
+		{
 			"ImportPath": "github.com/blang/semver",
-			"Comment": "v3.0.1",
-			"Rev": "31b736133b98f26d5e078ec9eb591666edfd091f"
+			"Comment": "v3.1.0",
```

Comment:
To keep align with runtime-tools which we are using now. https://github.com/opencontainers/runtime-tools/blob/master/Godeps/Godeps.json#L14-L16